tests/test_salesforce_incremental_table_reset.py

```diff
@@ -0,0 +1,83 @@
+import unittest
```

Comment:
There seems to be a lot of changes here.  The test went from 145 lines in commit f1689d398be8d0e704f1fd41fd5c2645c7cdd621. To Only 83 lines here.  There are at least 8 assertions in the previous test.  This one has only 2.

Reply:
Added back the needed assertions